zebedee-cms/src/main/java/com/github/onsdigital/zebedee/session/store/SessionsThreadLocalImpl.java

```diff
@@ -0,0 +1,72 @@
+package com.github.onsdigital.zebedee.session.store;
+
+import com.github.onsdigital.zebedee.session.store.exceptions.*;
+
+import com.github.onsdigital.exceptions.*;
+
+import com.github.onsdigital.impl.*;
+import com.github.onsdigital.interfaces.*;
+import javax.servlet.http.HttpServletRequest;
+import com.github.onsdigital.JWTHandlerImpl;
+
+/**
+ * This method creates a ThreadLocal for the validated jwt 
+ * from a HTTP request and appropriate key 
+ * as part of implementing dp-identity-api
+ */
+
+public class SessionsThreadLocalImpl implements SessionsThreadLocal {
+
+    private static HttpServletRequest request;
+    private static String secretKey;
+    private static ThreadLocal<UserDataPayload> store =
+            new ThreadLocal<>();
+
+    private JWTHandler jwtHandler = new JWTHandlerImpl();
+
+
+    public SessionsThreadLocalImpl(HttpServletRequest request, String secretKey) {
+        this.request = request;
+        this.secretKey = secretKey;
+    }
+
+    @Override
+    public void store(HttpServletRequest request, String secretKey ) throws SessionsStoreException {
+       /**
+        * Validates the Http Request header for null or empty 
+        * HttpServletRequest method for getHeader returns null if the request does not have a header of that name
+        */  
+
+        if ( request.getHeader("Authorization") == null || 
+                request.getHeader("Authorization").isEmpty() ) {
+                    throw new SessionsRequestException("Request does not have Authorization in Header");
+                };
```

Comment:
You don't need the `;` here also can  you run the formatter on this class please:

To format the class in intellij:
`opt` + `cmd` + `l`

To organise the imports:
`ctrl` + `opt` + `o`

Reply:
Can we refactor this to:
```
String token = request.getHeader("Authorization");
if (StringUtils.isEmpty(token) {
    // throw error
}
```
Commons lang isEmpty will check null and empty and I think it reads a bit nicer. Im being picky I know.